src/Controller/userController.js

```diff
@@ -0,0 +1,145 @@
+const axios = require("axios");
+const User = require("../Models/User");
+const env = require("../configs/dev");
+
+const login = async () => {
```

Comment:
What's the purpose of this function? No functionality implemented.

Reply:
this login function is just a callback function, just to test functionality but will be remove in next push.

Reply:
If it's not being used, it's not required. I'd like to avoid a bloated codebase.